### PR TITLE
Update endpoints with v16 and testnet migration

### PR DESCRIPTION
**Pull request by killerwhile**
Merge this only with the migration is effective.

This PR updates the doc to reflect the exact state of the endpoints for both mainnet and testnet.

### CODE REVIEW

File: docs/dapps/api-aliases.json [hunk 1]
```diff
@@ -2,93 +2,125 @@
...
+        },
         {
             "from": "https://wallet-v18.mainnet.alephium.org",
-            "to": "https://wallet-v15.mainnet.alephium.org"
+            "to": "https://wallet-v16.mainnet.alephium.org"
         },
         {
             "from": "https://wallet-v18.testnet.alephium.org",
-            "to": "https://wallet-v15.testnet.alephium.org"
+            "to": "https://wallet-v16.testnet.alephium.org"
```
Comment (nop33) [inline docs/dapps/api-aliases.json:51]: Thankfully there are no breaking changes between node 1.5.X and 1.6.X so this change will not cause a problem to users of the desktop wallet v1.4.0 (latest release).

File: docs/dapps/api-aliases.json [hunk 2]
```diff
@@ -2,93 +2,125 @@
...
         },
         {
             "from": "https://backend-v18.mainnet.alephium.org",
-            "to": "https://backend-v111.mainnet.alephium.org"
+            "to": "https://backend-v112.mainnet.alephium.org"
         },
         {
             "from": "https://backend-v18.testnet.alephium.org",
-            "to": "https://backend-v111.testnet.alephium.org"
+            "to": "https://backend-v112.testnet.alephium.org"
```
Comment (nop33) [inline docs/dapps/api-aliases.json:67]: Thankfully there are no breaking changes between explorer backend 1.8.X and 1.12.X so this change will not cause a problem to users of the desktop wallet v1.4.0 (latest release).

**Review by nop33 — APPROVED** [review]

File: docs/dapps/api-aliases.json (hunk 2 above)
Comment (killerwhile) [reply docs/dapps/api-aliases.json:67]: explorer backend 1.8.X already points to explorer backend 1.11.x, so the diff is between 1.11.x and 1.12.x, which I can confirm does not have any breaking changes.

File: docs/dapps/api-aliases.json [hunk 3]
```diff
@@ -2,93 +2,125 @@
...
             "from": "https://explorer.testnet.alephium.org",
-            "to": "https://explorer-v111.testnet.alephium.org"
+            "to": "https://explorer-v112.testnet.alephium.org"
+        },
+        {
+            "from": "https://explorer.alephium.org",
+            "to": "https://explorer-v112.mainnet.alephium.org"
+        },
+        {
+            "from": "https://testnet.alephium.org",
```
Comment (nop33) [inline docs/dapps/api-aliases.json:32]: @killerwhile I think this endpoint should be deprecated (redirected) in favor of `explorer.testnet.alephium.org`. The latter one should be the publicly visible endpoint. 
@mvaivre @killerwhile WDYT?

In this case, when the user uses the dropdown menu to switch between mainnet and testnet, they will see these 2 URLs on their browser:
- https://explorer.alephium.org
- https://explorer.testnet.alephium.org

Moreover, the desktop/mobile/extension wallet shall **always** point to the above 2 explorer endpoints.

**Comment by nop33** [comment]
When should this PR be merged? When the desktop wallet v1.5.0 is released?

File: docs/dapps/api-aliases.json (hunk 3 above)
Comment (mvaivre) [reply docs/dapps/api-aliases.json:32]: Yes, this would make sense.

**Comment by polarker** [comment]
> When should this PR be merged? When the desktop wallet v1.5.0 is released?

I will merge it right now, since the changes are already implemented.

**Pull request merged**